docs/content/en/docs/clustermgmt/cluster-upgrades/vsphere-and-cloudstack-upgrades.md

```diff
@@ -210,6 +210,29 @@ After finishing the task, make sure you resume the cluster reconciliation by rem
 kubectl annotate clusters.anywhere.eks.amazonaws.com ${CLUSTER_NAME} -n ${CLUSTER_NAMESPACE} anywhere.eks.amazonaws.com/paused-
 ```
 
+>**_NOTE (vSphere only):_** Installing and managing CSI as part of vSphere cluster operations was deprecated in EKS Anywhere version `v0.16.0` and will be removed in `v0.17.0`. If you are using EKS-A version v0.16.0 and above to upgrade a cluster that has the vSphere CSI Driver installed in it,
```

Comment:
Should we just link the section that talks about this instead of maintaining in two different places?

Reply:
I see that you just moved it. I think there are other places in the docs that references that `disableCSI` flag?